hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/SstFilteringService.java

```diff
@@ -187,9 +174,8 @@ public BackgroundTaskResult call() throws Exception {
           }
 
           // mark the snapshot as filtered by writing to the file
-          String content = snapshotInfo.getSnapshotId() + "\n";
-          Files.write(filePath, content.getBytes(StandardCharsets.UTF_8),
-              StandardOpenOption.CREATE, StandardOpenOption.APPEND);
+          snapshotInfo.setSstFiltered(true);
+          snapshotInfoTable.put(snapShotTableKey, snapshotInfo);
```

Comment:
Lock/Serializsation needed ? Other services could be changing the snapshot Info fields.

Reply:
Yeah realized locking is required here. Have updated the patch.

Reply:
done